### ISSUE TITLE
Changes in Emissions Reporting (high-level) for `climate-assessment` (FaIR, MAGICC) since AR6

### ISSUE THREAD

**Issue by jkikstra** [IAMconsortium/common-definitions#165]
**Topic:**
What subcomponents of CO2 are there? 
And how we need to update the climate-assessment workflow to accommodate the change in the template.

**AR6:**
In AR6, we had: 
Emissions|CO2 =
"Emissions|CO2|AFOLU" + "Emissions|CO2|Energy and Industrial Processes" + "Emissions|CO2|Other" + "Emissions|CO2|Waste"

this is also how the current climate-assessment code ([docs](https://climate-assessment.readthedocs.io/en/latest/index.html)) is working.

**ScenarioMIP**
In the new template ([I am looking here](https://github.com/IAMconsortium/common-definitions/blob/main/definitions/variable/emissions/emissions.yaml)), this does not hold true anymore. 

In addition to the four above, we now also have:
“Fossil Fuel Fires”, “Capture and Removal”, “Product Use”

Note also, that at the top of "[emissions.yaml](https://github.com/IAMconsortium/common-definitions/blob/main/definitions/variable/emissions/emissions.yaml)", we have the following, where "Product Use", and "Fossil Fuel Fires" are missing:
- Emissions|{Level-1 Species}:
    description: Emissions of {Level-1 Species}
    unit: "{Level-1 Species}"
    components:
      - Emissions|{Level-1 Species}|AFOLU
      - Emissions|{Level-1 Species}|Energy
      - Emissions|{Level-1 Species}|Industrial Processes
      - Emissions|{Level-1 Species}|Waste
      - Emissions|{Level-1 Species}|Capture and Removal
      - Emissions|{Level-1 Species}|Other

**How to go forward?**

Option 1: revert back to AR6 aggregate template.
Option 2: agree on a full definition of (a) which sectors to report, and (b) clarify that these components together do add up to the total. Then also, `climate-assessment` will have to be updated accordingly.

**Questions:**
For option 2: should we go with the full set? 
I.e.: 
- Emissions|{Level-1 Species}:
    description: Emissions of {Level-1 Species}
    unit: "{Level-1 Species}"
    components:
      - Emissions|{Level-1 Species}|AFOLU
      - Emissions|{Level-1 Species}|Energy
      - Emissions|{Level-1 Species}|Industrial Processes
      - Emissions|{Level-1 Species}|Waste
      - Emissions|{Level-1 Species}|Capture and Removal
      - Emissions|{Level-1 Species}|Other
      - Emissions|{Level-1 Species}|Fossil Fuel Fires
      - Emissions|{Level-1 Species}|Product Use

In FaIR and MAGICC, we only have two sectors in CO2, and 1 sector for all other species.
For CO2, should we go with:
(1) AFOLU; Emissions|CO2|AFOLU
(2) Energy and Industrial Processes; sum of all other sectors.

@IAMconsortium/common-definitions-emissions 
@znicholls @chrisroadmap @gidden 

_Related issues:_
* Are CDR descriptions clear enough? #145 
* Updating docs of `climate-assessment`: https://github.com/iiasa/climate-assessment/issues/61
* Discussion on variables to report for harmonization of emissions for ScenarioMIP/CMIP7: #151

**Comment by znichollscr** [comment]
> In FaIR and MAGICC, we only have two sectors in CO2, and 1 sector for all other species.
> For CO2, should we go with:
> (1) AFOLU; Emissions|CO2|AFOLU
> (2) Energy and Industrial Processes; sum of all other sectors.

This is all MAGICC and FaIR have, so all we need/can use. The idea is basically: split CO2 emissions into CO2 that came from fossil reservoirs vs. CO2 that came from the land pool. As long as we get something that has roughly that split, we'll be fine, so you can do whatever you want at the more detailed levels below this binary split.

**Comment by jkikstra** [comment]
> > In FaIR and MAGICC, we only have two sectors in CO2, and 1 sector for all other species.
> > For CO2, should we go with:
> > (1) AFOLU; Emissions|CO2|AFOLU
> > (2) Energy and Industrial Processes; sum of all other sectors.
> 
> This is all MAGICC and FaIR have, so all we need/can use. The idea is basically: split CO2 emissions into CO2 that came from fossil reservoirs vs. CO2 that came from the land pool. As long as we get something that has roughly that split, we'll be fine, so you can do whatever you want at the more detailed levels below this binary split.

That's right!

On the IAM side we just need to make sure that we're actually harmonizing to more detailed totals! (And that these totals indeed add up alright.)
So people will want to know where to report their CDR emissions (see also e.g. https://github.com/IAMconsortium/common-definitions/issues/145).
And if someone has fossil fire emissions, whether to report it under 'Energy' or directly under 'Fossil Fuel Fires'.
It's just for the template, even if in `climate-assessment` it will have no impact (we just need to reaggregate as a pre-processing step) - as long as there's no mixed AFOLU/Fossil emissions category being introduced, which is currently not the case I think.

**Comment by jkikstra** [comment]
In the first submission, none of the modelling teams reported "Fossil Fuel Fires".

Unless anyone shouts (@IAMconsortium/common-definitions-emissions), I suggest we move it under "Energy" as 
"Emissions|{species}|Energy|Fossil Fuel Fires"

**Comment by znicholls** [comment]
Just checking the definition: "Fossil fuel fires are fires based on burning of fossil fuels, lit by people" (or something like this)? If yes, sounds good because a) putting it under energy means it'll go in the 'fossil' bucket for MAGICC and FaIR and b) it's clearly anthropogenic (so none of the natural vs. anthropogenic split headache).

**Comment by jkikstra** [comment]
Maybe it has been reported by (some?) IAMs before, but I am not aware of that. And for CMIP6, for emissions harmonization, it was classified under "Energy Sector" too. 

Looks like it was added as a separate category because of CEDS ([git blame](https://github.com/IAMconsortium/common-definitions/commit/258c943fdbc16f362865367dba68daba50bcac93)).
In CEDS, it is reported under "7A" (referring to IPCC 'other'), I'm not entirely sure why ([CEDS wiki](https://github.com/JGCRI/CEDS/wiki/Data_and_Assumptions) is not very clear about it). Perhaps it is to because they used a separate gridding proxy data file? (EDGAR v4.2 FFFI)

Note also that in [Hoesly et al.](https://gmd.copernicus.org/articles/11/369/2018/gmd-11-369-2018.pdf) it is also classified under the Energy sector (Table 6).

**Comment by znicholls** [comment]
Perfect. Sounds like everything is pointing in the direction of your solution being a good one!

**Comment by jkikstra** [comment]
Yeah - will all shoot off a quick email to @ssmithClimate just out of curiosity on why CEDS reports it like this, then we should be sure.

**Comment by jkikstra** [comment]
> Unless anyone shouts (@IAMconsortium/common-definitions-emissions), I suggest we move it under "Energy" as "Emissions|{species}|Energy|Fossil Fuel Fires"

Actually, tiny update, it probably should be "Emissions|{species}|Energy|**Supply**|Fossil Fuel Fires"

**Comment by ssmithClimate** [comment]
It doesn’t really fit either one. 

It’s not fugitive emissions, so it doesn’t really fit under 1B/2B. 

But it’s not really combustion from energy energy, production, or transformation. 

It’s more like a third category of “open burning “, that happens to be fossil fuels.

Which is why it’s put in an “other” category by Edgar, which we also agree with. 

So the main issue perhaps, given that I suspect few models include this, from an IAMC perspective, how do we properly harmonize this with IAM data? Will it be clear to the processing that IAMs are not including this, and therefore, needs to be gap filled? (Really a more general issue.)

**Comment by danielhuppmann** [comment]
Two comments: from my basic intuition as an IAM modeler, adding fossil-fuel as a subcategory of energy-supply is not very intuitive. And if this is going to be a variable that few IAMs can even report (or have a detailed implementation for), I’d prefer to keep it close to the top of the variable tree. With the explicit aim that as many variables as possible are close to reported model-values after harmonization, or keeping major differences before and after harmonization to a few variables.

**Comment by jkikstra** [comment]
> Two comments: from my basic intuition as an IAM modeler, adding fossil-fuel as a subcategory of energy-supply is not very intuitive. And if this is going to be a variable that few IAMs can even report (or have a detailed implementation for), I’d prefer to keep it close to the top of the variable tree. With the explicit aim that as many variables as possible are close to reported model-values after harmonization, or keeping major differences before and after harmonization to a few variables.

Then what about under "Emissions|CO2|Other", and add it explicitly to the definition there?
Also aligns well between with @ssmithClimate's comment? 

I'm just trying to avoid us having a high-level variable that nobody will report. 






However, if you strongly object, we can keep it like it
